pyop/pykernel.h

```diff
@@ -60,10 +60,6 @@ struct PyCustomOpFactory : OrtW::CustomOpBase<PyCustomOpFactory, PyCustomOpKerne
     op_type_ = op;
   }
 
-  void* CreateKernel(const OrtApi& api, const OrtKernelInfo* info) const {
-    return CreateKernelImpl(api, info, opdef_->obj_id, opdef_->attrs);
```

Comment:
there are some extra arguments here (opdef_->obj_id...), doesn't this removal cause some issues?

Reply:
Good catch. Fixed.